manual/cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -283,3 +283,24 @@ include::split.asciidoc[leveloffset=+1]
 include::reverse.asciidoc[leveloffset=+1]
 
 include::tostring.asciidoc[leveloffset=+1]
+
+ifndef::asciidoctor[:leveloffset: 1]
+
+[[query-functions-udf]]
+== User-defined functions ==
+User-defined functions are written in Java, deployed into the database and are called in the same way as any other Cypher function.
```

Comment:
Additionally, I'd like to suggest a second paragraph:

UDFs share many similarities with procedures, but they differ in two key ways: they are read-only and may only return scalar values. These constraints makes it possible for the Cypher planner to treat them just as if they were built-in functions.